test/e2e/config/e2e_conf.yaml

```diff
@@ -302,6 +302,7 @@ variables:
   IRONIC_RELEASE_27.0: "data/ironic-deployment/overlays/release-27.0"
   IRONIC_RELEASE_29.0: "data/ironic-deployment/overlays/release-29.0"
   IRONIC_RELEASE_31.0: "data/ironic-deployment/overlays/release-31.0"
+  IRONIC_RELEASE_32.0: "data/ironic-deployment/overlays/release-32.0"
```

Comment:
We need to add this version also in the clusterctl upgrade test, e.g. here: https://github.com/metal3-io/cluster-api-provider-metal3/blob/d6b1bcc25749f7a536c1704e359067524603f7d5/test/e2e/upgrade_clusterctl_test.go#L54
It is used to install Ironic from this kustomization when starting the test: https://github.com/metal3-io/cluster-api-provider-metal3/blob/d6b1bcc25749f7a536c1704e359067524603f7d5/test/e2e/upgrade_clusterctl_test.go#L477

Reply:
Thanks, Will add it